knative-operator/cmd/manager/main.go

```diff
@@ -124,6 +125,16 @@ func main() {
 
 	log.Info("Starting the Cmd.")
 
+	go func() {
+		// This web server is unimportant enough to not bother connecting its lifecycle to
+		// signal handling so the process can just tear it down with it.
+		log.Info("Serving CLI artifacts on :8080")
+		http.Handle("/", http.FileServer(http.Dir("/cli-artifacts")))
+		if err := http.ListenAndServe(":8080", nil); err != nil && err != http.ErrServerClosed {
```

Comment:
Should we protect this with rbac-proxy?

Reply:
See above, I don't think any auth is necessary here.